app/models/export.rb

```diff
@@ -65,6 +65,22 @@ def export_object_filepath(message_object)
     out.sub(/^\/+/, '')
   end
 
+  def self.form_name(object, include_version: true)
+    form = Fs::Form.find_by(id: object.message.thread.metadata["fs_form_id"])
+    return unless form
+
+    prefix = case form.subtype_name
+             when "Dodatočné"
+               "DOD"
+             when "Opravné"
+               "OPR"
+             end
```

Comment:
Pozor, napr. SVDPHv20 má (aspoň v FS API, neviem, či je to nejak inak transformované v GO) viac možností:
```
Dodatočný
Dodatočný Zástupca pri dovoze
Opravný
Opravný Zástupca pri dovoze
Riadny
Riadny Zástupca pri dovoze
```

Reply:
Zatial by som zostala iba pri tomto, ak sa tieto dalsie pripady vyskytnu a bude poziadavka ich nejako specialne pomenovat, zistime aspon ake pomenovanie sa ocakava a zapracujeme.

Reply:
nedáme tam rovno nejaké `DOD_ZAST`, `OPS_ZAST`?

Reply:
podla mna nie, take podania zatial nasi klienti ani nerobia